backend/src/v5/models/roles.js

```diff
@@ -28,4 +47,41 @@ Roles.grantTeamspaceRoleToUser = (teamspace, username) => db.grantRole(teamspace
 
 Roles.revokeTeamspaceRoleFromUser = (teamspace, username) => db.revokeRole(teamspace, TEAM_MEMBER, username);
 
+// --- JOBS RENAMED TO ROLES --- TO BE KEPT
+
+Roles.getRolesToUsers = (teamspace) => findMany(teamspace, {}, { _id: 1, users: 1 });
+
+Roles.getRoles = (teamspace, projection) => findMany(teamspace, {}, projection);
+
+Roles.getRoleById = (teamspace, roleId, projection) => findOne(teamspace, { _id: roleId }, projection);
```

Comment:
Unless linter hates it, we can just do this? 
```suggestion
Roles.getRoleById = (teamspace, _id, projection) => findOne(teamspace, { _id }, projection);
```